src/test/java/org/traccar/protocol/Gt06ProtocolDecoderTest.java

```diff
@@ -334,12 +334,21 @@ public void testDecode() throws Exception {
                 Position.KEY_ALARM, Position.ALARM_ACCELERATION);
 
         verifyAttribute(decoder, binary(
-                "78780c95130a0209321c90000112800d0a"),
-                "alarmValue", 1);
+                "78780c95130a0209321c91000112800d0a"),
```

Comment:
What is the reason for changing existing test cases?

Reply:
It is a git merge thing - I removed the `alarmValue` above, also removed the test case. Then added new test fro JC400 braking. It looks like I have changed the test case in diff but it is not.